crimethinc/website
# Bump ruby version to ruby 3.1

# How does this pull request make you feel (in animated GIF format)?

![excited person](https://media.giphy.com/media/UO5elnTqo4vSg/giphy.gif)

# What does this pull request do?

It upgrades ruby to 3.1.0

# How should this be manually tested?

* Check that all the tests pass
* Check on the staging environment that everything seems to work
